platform/MCU/NRF51/SPARK_Firmware_Driver/src/hw_config.c

```diff
@@ -146,6 +147,9 @@ void FLASH_Begin(uint32_t sFLASH_Address, uint32_t fileSize)
 {
     //OTA_FLASHED_Status_SysFlag = 0x0000;
     //Save_SystemFlags();
+#if PLATFORM_ID==103
```

Comment:
should that be:
for (int i=1; i<APP_TIMER_MAX_TIMERS; i++) app_timer_stop(i);

??